pkg/ccl/changefeedccl/avro_test.go

```diff
@@ -479,21 +480,24 @@ func TestAvroSchema(t *testing.T) {
 		for _, typ := range append(types.Scalar, types.BoolArray, types.MakeCollatedString(types.String, `fr`), types.MakeBit(3)) {
 			switch typ.Family() {
 			case types.OidFamily:
-				continue
+				return
```

Comment:
This diff looks like it short-circuits the test as soon as we hit an OID type?

Reply:
ah a bad edit - thanks!